### PR TITLE
add philly config

### LINKED ISSUE #2366: Create a "Philadelphia" configuration for the orchestration service
## What needs to be done

Create a new configuration for the orchestration service to support the Philadelphia version of the eCR Viewer.

## Why it needs to be done

Philly wants an extension to the core eCR Viewer functionality - they'd also like to be able to query a database of eCR data. We need to ensure that we can support that functionality.

## Acceptance Criteria

-  [x] Create a new configuration, `philadelphia-ecr-viewer`
-  [ ] Config should include the following steps:
    -  [x] Conversion to FHIR
    -  [x] Standardize Name
    -  [x] Standardize DOB
    -  [x] Standardize Phones
    -  [x] Stamp eCR
    -  [x] Message Parser Values (with Philly config developed in #)
    -  [ ] Save data to eCR Viewer 

## Technical Notes

The existing `seed-ecr-viewer-config` covers most of this functionality; it's the message parser type and save to eCR Viewer that will need to be changed.

### PR DESCRIPTION
**Pull request by lina-roth**
# PULL REQUEST

Screen Shot 2024-08-29 at 4 02 06 PM

## Summary
- Add Philly config based on ticket

## Related Issue
Fixes #2366 

## Acceptance Criteria
 Create a new configuration, philadelphia-ecr-viewer
 Config should include the following steps:
 Conversion to FHIR
 Standardize Name
 Standardize DOB
 Standardize Phones
 Stamp eCR
 Message Parser Values (with Philly config developed in #)
 Save data to eCR Viewer
 
## Additional Information
Anything else the review team should know?

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

File: containers/orchestration/app/custom_configs/philadelphia-ecr-viewer.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        "credential_manager": "azure"
+      }
+    },
+    {
+      "name:": "save_bundle",
+      "service": "save_bundle",
+      "url": "${ECR_VIEWER_URL}",
+      "endpoint": "/api/save-fhir-data",
+      "params": {
+        "saveSource": "postgres"
```
Comment (BobanL) [inline containers/orchestration/app/custom_configs/philadelphia-ecr-viewer.json:44]: Do we want the saveSource to be postgres for philly or should it be s3? @gordonfarrell

**Comment by BobanL** [comment]
Don't know why but locally I wasn't able to get passed message parser step. Does anyone know why?

image
image

**Comment by angelathe** [comment]
@lina-roth Tested with @BobanL and @austin-hall-skylight in a huddle, and I think the Philly config needs to switch the order of message parser and save-fhir-data (following what `sample-orchestration-config.json` did) -- that got rid of my errors

File: containers/orchestration/app/custom_configs/philadelphia-ecr-viewer.json (hunk 1 above)
Comment (gordonfarrell) [reply containers/orchestration/app/custom_configs/philadelphia-ecr-viewer.json]: Yup saveSource should be s3 for Philly! Nice catch

**Review by angelathe — APPROVED** [review]
LGTM!

**Pull request merged**